### ISSUE TITLE
Algorithms with code

### ISSUE THREAD

**Issue by spono** [Jean-Romain/lidRplugins#21]
Ciao JR,
looking for algos related to LiDAR analyses, I found the following ones providing also the code. Unfortunately it's MATLAB but I saw that -having its IDE- there's the possibility to export/convert code to C/C++, but I don't know if/how this could help its integration in `lidR`.

1. [detection of fallen trees](https://github.com/Eikka12/Fallen_tree_detection/blob/main/main/findFallenTrees.m) (Heinaro et  al. 2021)
2. [segmentation according to point density](https://zenodo.org/records/4421030) (Latella et al., 2021)

Don't take this message as a request: I only hope it may help to further improve the great package(s) you created :)
HTH!

**Comment by spono** [comment]
[another one](https://opus4.kobv.de/opus4-hnee/frontdoor/index/index/docId/372) posted from @wiesehahn in [lidR issues](https://github.com/r-lidar/lidR/issues/736), this time in R/C++.

**Comment by Jean-Romain** [comment]
`lidRpluging` is an abandoned project. I appreciate your contribution but there is little chance that I add anything in this repo.

**Comment by spono** [comment]
oh, ok, I didn't know it was abandoned: sorry to hear! it was full of interesting tools to play with :)
BTW, thanks again!

**Comment by Jean-Romain** [comment]
I'm not even sure if lidRplugins still works and is compatible with lidR v4+ haha. Anyway I'm opened to suggestion. But as I said it require an article with either a great description and pseudo code or an open source code. Coding an algorithm from a paper requires weeks and often it actually lead to something that worked only for the authors in their specific point cloud. It is not something I can do.

**Comment by spono** [comment]
> Coding an algorithm from a paper requires weeks and often it actually lead to something that worked only for the authors in their specific point cloud.

I know...and the pain is quite high! that's why I posted only these algos which provided real code: maintaining a package like `lidR` is already tough, there's no need for further stress :) 

> I'm not even sure if lidRplugins still works and is compatible with lidR v4+

as soon as I'll have time to test again some of the package tools (the ones related to transmission towers are vital, in some cases) I'll check and try to contribute as I can

> Anyway I'm opened to suggestion

no real package-related suggestion, right now...but let's not waste al the good work you put into this side-project :)

**Comment by Jean-Romain** [comment]
Have you seen my latest project lasR? https://github.com/r-lidar/lasR That one is in the r-lidar repository and is not an experimental project.

**Comment by spono** [comment]
yep, I tested it mostly in its first steps: another great piece of sw!
nevertheless, having a limited amount of time for lidar processing and having all my workflows ready in lidR, I mostly interested in your side-projects like ALSroads, ALSbridges and the tools related to powerlines included in this package :)
